composer.json

```diff
@@ -8,11 +8,11 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1",
         "sonata-project/admin-bundle": "^3",
-        "symfony/yaml": "^3.4",
-        "symfony/finder": "^3.4",
-        "doctrine/doctrine-bundle": "^1.6",
+        "symfony/yaml": "^4.4",
```

Comment:
Kan aan mij liggen, maar ik zie niet heel veel dat niet op 3.4 zou draaien, of heb ik het mis?